subsys/bluetooth/audio/ascs.c

```diff
@@ -1136,8 +1124,30 @@ static void ase_init(struct bt_ascs_ase *ase, uint8_t id)
 	k_work_init(&ase->work, ase_process);
 }
 
+static void audio_iso_init(struct bt_audio_iso *audio_iso)
+{
+	/* Setup points for both sink and source
+	 * This is due to the limitation in the ISO API where pointers like
+	 * the `qos->tx` shall be initialized before the CIS is connected if
+	 * ever want to use it for TX, and ditto for RX. They cannot be
+	 * initialized after the CIS has been connected
+	 */
+	audio_iso->iso_chan.ops = &ascs_iso_ops;
+	audio_iso->iso_chan.qos = &audio_iso->iso_qos;
+	audio_iso->iso_chan.qos->tx = &audio_iso->source_io_qos;
+	audio_iso->iso_chan.qos->tx->path = &audio_iso->source_path;
+	audio_iso->iso_chan.qos->tx->path->cc = audio_iso->source_path_cc;
+
+	audio_iso->iso_chan.ops = &ascs_iso_ops;
```

Comment:
initialized already in line 1135

Reply:
Thanks fixed. Moved the lines so it's clearer what is going on.